File: archives/python/5086.py
import sys

while 1:
    a, b = map(int, sys.stdin.readline().strip().split())
    if a == 0 and b == 0: break
    else:
        if a >= b:
            if a % b == 0: print("multiple")
            else : print("neither")
        else:
            if b % a == 0 : print("factor")
            else : print("neither")

File: archives/cpp/5086.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);
    
    int A, B;
    while (true) {
        cin >> A >> B;
        if (A == 0 && B == 0) { break; }

        if (A % B == 0) { cout << "multiple\n"; }
        else if (B % A == 0) { cout << "factor\n"; }
        else { cout << "neither\n"; }
    }

    return 0;
}